azurerm/helpers/azure/servicebus.go

```diff
@@ -28,7 +28,7 @@ func ValidateServiceBusQueueName() schema.SchemaValidateFunc {
 
 func ValidateServiceBusSubscriptionName() schema.SchemaValidateFunc {
 	return validation.StringMatch(
-		regexp.MustCompile("^[a-zA-Z][-._a-zA-Z0-9]{0,48}([a-zA-Z0-9])?$"),
+		regexp.MustCompile("^[a-zA-0-9][-._a-zA-Z0-9]{0,48}([a-zA-Z0-9])?$"),
```

Comment:
I think this should be
```suggestion
		regexp.MustCompile("^[a-zA-Z0-9][-._a-zA-Z0-9]{0,48}([a-zA-Z0-9])?$"),
```